app/src/Pages/Home/Home.js

```diff
@@ -1056,7 +1057,7 @@ const Home = ({
                   campaign => campaign.campaignName === name
                 );
                 return (
-                  <React.Fragment key={name}>
+                  <React.Fragment key={`candidate-${name}-index`}>
```

Comment:
nit: i think you should avoid indices in a react key.

this is better than a pure index key because the name will distinguish it across data, but if the underlying list changes react will unnecessarily rerender this component just because the index is different

Reply:
I hear you and agree indices should not be used when naming keys. This however is not using the dynamic index, but rather is relying on the `${name}` to be the differentiator. The name + additional info helps me when using the react debugger. Bueno?